app/src/main/java/de/triology/cas/logging/DefaultMappingPasswordRewritePolicy.java

```diff
@@ -33,7 +33,7 @@ protected String getPasswordFlag() {
     protected String replacePasswordValue(String originMessage) {
         String modifiedMessage = null;
         if (originMessage != null) {
-            modifiedMessage = originMessage.replaceAll("mappedValue = '\\S+'", "mappedValue = '****'");
+            modifiedMessage = originMessage.replaceAll("mappedValue = array<Character>\\[.*\\]", "mappedValue = '****'");
```

Comment:
A second thing: This regex would not match the line in the supportticket, would it?
"mappedValue = array[T, e, s, t]" does not contain the `<Character>` part.

Reply:
I didn't get this one but I do concur in that it seems that we do not need this line at all.

Thus I suggest something along the lines of this so we do not have to build pattern matching during runtime

```java
public final class DefaultMappingPasswordRewritePolicy extends AbstractCASPasswordRewritePolicy {
    private static final String PARAMETER_PASSWORD_TEXT = "parameter:'password'";
    private static final Pattern passwordDebugLogLine = Pattern.compile("^.+" + PARAMETER_PASSWORD_TEXT + ".+$");
    private static final String replaceMessage = "this log line contained sensitive data and was removed";

// later in this class

    protected String replacePasswordValue(String originMessage) {
        if (originMessage == null || originMessage.isEmpty()) {
            return null;
        }

        boolean logLineContainsPassword = passwordDebugLogLine.matcher(originMessage).find();
        if (logLineContainsPassword) {
            return replaceMessage;
        }

        return originMessage;
    }
```

Reply:
I think that is good too. But we do not have to match the `PARAMETER_PASSWORD_TEXT` again. The `AbstractCASPasswordRewritePolicy` checks this with `containsPassword(LogEvent source)`